relay-general/src/protocol/types.rs

```diff
@@ -1117,6 +1117,26 @@ impl schemars::JsonSchema for Timestamp {
     }
 }
 
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue, ProcessValue)]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct DataElement {
+    #[metastructure(pii = "maybe")]
+    pub http: Annotated<HttpElement>,
+
+    #[metastructure(additional_properties, retain = "true", pii = "maybe")]
+    pub other: Object<Value>,
+}
+
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue, ProcessValue)]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct HttpElement {
+    #[metastructure(pii = "true")]
+    pub query: Annotated<String>,
```

Comment:
`span.data` is [documented as free-form](https://develop.sentry.dev/sdk/event-payloads/span/), so when a user sends `{"http": {"query": {"not": "a string"}}}`, we should still accept it.

@iker-barriocanal could you check if the scrubbing still works when you change `String` to `Value` here?

Reply:
Great catch! This is fixed in 03af9eb46153a99ecada9db886da4c6dab5ffbaf, and continues to work as expected.